crypto/evp_extra/evp_asn1.c

```diff
@@ -91,18 +91,27 @@ static const EVP_PKEY_ASN1_METHOD *parse_key_type(CBS *cbs) {
   if (OBJ_cbs2nid(&oid) == NID_rsa) {
     return &rsa_asn1_meth;
   }
+
+#ifdef ENABLE_DILITHIUM
+  // if |cbs| is empty we overwrite the contents with |oid| so that we can
```

Comment:
Nit:
```Suggestion
  // if |cbs| is empty (after parsing |oid| from it), we overwrite the contents with |oid| so that we can
```
(line length will need adjustment)

Reply:
fixed in [c71bbac](https://github.com/aws/aws-lc/pull/1963/commits/c71bbac49160038ba8262b9f1996b43e197410ba)